### PR TITLE
Add connector (api_inspectionplan_srv)  

### PR DESCRIPTION
**Pull request by EsalaMapa**
## Purpose

Fixes:$subject

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: ballerina/api_inspectionplan_srv/Module.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+## Overview
+
+[S/4HANA](https://www.sap.com/india/products/erp/s4hana.html) is a robust enterprise resource planning (ERP) solution,
+designed for large-scale enterprises by SAP SE.
+
+The `ballerinax/sap.s4hana.api_inspectionlot_srv` package offers APIs for seamless integration with the [Inspection Lot API v1.2.0](https://api.sap.com/api/API_INFORECORD_PROCESS_SRV/overview).In the SAP system, you process quality inspections on the basis of inspection lots. The data relating to a quality inspection is grouped into several entities. With this service it is possible, for example, to update inspection lot header data or to create inspection results and usage decisions.
```
Comment (niveathika) [inline ballerina/api_inspectionplan_srv/Module.md]: Is the link to API documentation correct?

File: ballerina/api_inspectionplan_srv/Module.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Quickstart
+
+To use the `sap.s4hana.api_inspectionlot_srv` connector in your Ballerina application, modify the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `sap.s4hana.api_inspectionlot_srv` module.
+
+```ballerina
+import ballerinax/sap.s4hana.api_inspectionlot_srv as inspectionlot
```
Comment (niveathika) [inline ballerina/api_inspectionplan_srv/Module.md]: There should be semicolon in the end

File: ballerina/api_inspectionplan_srv/Module.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+);
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+
+```ballerina
+inspectionlot:Wrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
```
Comment (niveathika) [inline ballerina/api_inspectionplan_srv/Module.md]: Inspection lot/ inspection plan? Review the docs.json. file fully

File: ballerina/api_inspectionplan_srv/build.gradle [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+plugins {
+    id 'net.researchgate.release'
+    id 'io.ballerina.plugin'
+}
+
+description = 'Ballerina - SAP S/4HANA Inspection Plan API'
+
+def packageName = "sap.s4hana.api_inspectionplan_srv"
+def moduleVersion = project.api_inspectionlot_srvVersion.replace("-SNAPSHOT", "")
```
Comment (niveathika) [inline ballerina/api_inspectionplan_srv/build.gradle]:
```suggestion
def moduleVersion = project.api_inspectionplan_srvVersion.replace("-SNAPSHOT", "")
```

File: ballerina/api_inspectionplan_srv/build.gradle [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+            }
+        }
+    }
+}
+
+release {
+    buildTasks = ['build']
+    failOnSnapshotDependencies = true
+    versionPropertyFile = '../../gradle.properties'
+    versionProperties = ['api_inspectionlplan_srvVersion']
```
Comment (niveathika) [inline ballerina/api_inspectionplan_srv/build.gradle]:
```suggestion
    versionProperties = ['api_inspectionplan_srvVersion']
```

File: changelog.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## sap.s4hana.api_inspectionlot_srv
+## sap.s4hana.api_inspectionplan_srv
```
Comment (niveathika) [inline changelog.md]: This should be added to end of file as spearate heading with its own sub headings

File: ballerina/api_inspectionplan_srv/Module.md (hunk 1 above)
Comment (EsalaMapa) [reply ballerina/api_inspectionplan_srv/Module.md]: Please find fix in [Update docs.json in two modules to fix automated document generation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/10c211e4570ea2d2501780ca1029d936b69f092b)

File: ballerina/api_inspectionplan_srv/Module.md (hunk 2 above)
Comment (EsalaMapa) [reply ballerina/api_inspectionplan_srv/Module.md]: Please find fix in [Update docs.json in two modules to fix automated document generation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/10c211e4570ea2d2501780ca1029d936b69f092b)

File: ballerina/api_inspectionplan_srv/Module.md (hunk 3 above)
Comment (EsalaMapa) [reply ballerina/api_inspectionplan_srv/Module.md]: Please find fix in [Update docs.json in two modules to fix automated document generation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/10c211e4570ea2d2501780ca1029d936b69f092b)

File: ballerina/api_inspectionplan_srv/build.gradle (hunk 4 above)
Comment (EsalaMapa) [reply ballerina/api_inspectionplan_srv/build.gradle]: Fixed in here [Update changelog.md and build.gradle](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/6f7366330a9d72ccd12261b35117c8fff29e28a3)

File: ballerina/api_inspectionplan_srv/build.gradle (hunk 5 above)
Comment (EsalaMapa) [reply ballerina/api_inspectionplan_srv/build.gradle]: Fixed in here [Update changelog.md and build.gradle](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/6f7366330a9d72ccd12261b35117c8fff29e28a3)

File: changelog.md (hunk 6 above)
Comment (EsalaMapa) [reply changelog.md]: Fixed in here [Update changelog.md and build.gradle](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/6f7366330a9d72ccd12261b35117c8fff29e28a3)

File: ballerina/api_inspectionlot_srv/Package.md [hunk 7]
```diff
@@ -70,7 +70,7 @@ inspectionlot:Client inspectionlotClient = check new (
 Now, utilize the available connector operations.
 
 ```ballerina
-inspectionlot:Wrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
+inspectionlot:A_InspectionLotWrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
```
Comment (niveathika) [inline ballerina/api_inspectionlot_srv/Package.md]: Doesnt the create require payload? Wont list be better?

Comment (EsalaMapa) [reply ballerina/api_inspectionlot_srv/Package.md]: Updated here [Update docs.json in both modules](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/c21a6a16b9b47fb17256167ac2be9a83b289f8c6)

**Review by niveathika — APPROVED** [review]

**Pull request merged**